docs/docs/reference/dropped/weak-conformance-spec.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: doc-page
+title: Dropped: Weak Conformance - More Details
+---
+
+To simplify the underlying type theory, Dotty drops the notion of weak
+conformance altogether. Instead, it provides more flexibility when
```

Comment:
I just realized that in the spec, the definition of numeric widening depends on the definition of weak conformance, from https://www.scala-lang.org/files/archive/spec/2.12/06-expressions.html:
> If e has a primitive number type which weakly conforms to the expected type, it is widened to the expected type using one of the numeric conversion methods toShort, toChar, toInt, toLong, toFloat, toDouble defined here.

So we'll need to rewrite that definition (relatedly, I'm also trying to restrict what numeric widening can do in https://github.com/scala/scala/pull/7405)

Reply:
Actually it seems that we simply don't have any notion of numeric widening in Dotty, we just rely on the implicit conversions defined in Int.scala/Float.scala/etc